odometry_node:
  ros__parameters:
    # tf tree configuration
    base_frame: ""
    odom_frame: odom
    publish_odom_tf: true

    #--- KISS-ICP configuration
    # config.data
    deskew: false
    max_range: 100.0
    min_range: 5.0

    # config.mapping
    voxel_size: 1.0
    max_points_per_voxel: 20

    # config.adaptive_threshold
    initial_threshold: 2.0
    min_motion_th: 0.1
